community/asset_library/submitting_to_assetlib.rst

```diff
@@ -86,18 +86,13 @@ library a better place for all users.
 
   .. code-block:: none
 
-    # Normalize EOL for all files that Git considers text files.
+    # Normalize line endings for all files that Git considers text files.
     * text=auto eol=lf
 
-    # Ignore some files when exporting to a ZIP.
-    /.gitattributes     export-ignore
-    /.gitignore         export-ignore
-    /LICENSE            export-ignore
-    /LICENSE.md         export-ignore
-    /README.md          export-ignore
-    /project.godot      export-ignore
-    /icon.png           export-ignore
-    /icon.svg           export-ignore
+    # For addons, only include the addons folder. This is not needed for asset packs or projects.
```

Comment:
```suggestion
    # For addons, only include the addons folder when exporting a ZIP.
    # These lines should be deleted for asset packs or projects.
```

Reply:
I don't think the average asset maker knows that the asset library uses GitHub ZIPs internally.

Reply:
Attempt 2:

```suggestion
    # For addons, only include the addons folder when downloading from the Asset Library.
    # These lines should be deleted for asset packs or projects.
```